stories/link-bars/MatTagBarStory.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { DemoContainer } from '../storybook-components';
+
+import { Heading, HugeHeading } from '../../src/atoms/Heading';
+import { Code } from '../../src/atoms/Code';
+
+// HorizontalLinkBar
+import {
+	ScrollArrowsLinkBar,
+	LinkBarLinkBordered,
+} from '../../src';
+
+import theme from '../../src/themes/mat';
+
+const { colors } = theme;
+
+const MatTagBarStory = () => (
+	<section>
+		<HugeHeading>ScrollArrowsLinkBar</HugeHeading>
+		<p>A horizontal navbar that will be used for tags in MAT project. </p>
+
+		<Heading>Demo</Heading>
+		<DemoContainer background={colors.white}>
+			<ScrollArrowsLinkBar
+				background={colors.white}
+				arrowColor={colors.primary}
```

Comment:
When introducing new color props, please send color names that exists on the colors object, like this:

```
arrowColor="primary"
```

In ScrollArrowsLinkBar, we would then do something like this
```js
import { getColor } from '../utils';

const Arrow = styled(something)`
    color: ${props => getColor(props.arrowColor)
`
```

Reply:
(On the HorizontalLinkBar, ScrollArrowsLinkBar and so on, the background prop will be refactored into this way as soon as I can find the time)